css-brochure-form/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>LEARN MORE</h1>
+
+    <form class="learnForm">
+      <div class="learnForm">
+        <label for="email" class="heading">COMPANY EMAIL</label>
+      </div>
+      <div class="learnForm">
+        <input
+          type="email"
+          class="input"
+          id="email"
+          placeholder="sally@company.co"
+          required />
+      </div>
+
+      <div class="learnForm">
+        <label for="name" class="heading">NAME</label>
+      </div>
+      <div class="learnForm">
+        <input
+          type="text"
+          class="input"
+          id="username"
```

Comment:
Either your `label`s `for` needs to become `username` or your `input`s `id` needs to become `name`